compiler/rustc_middle/src/infer/unify_key.rs

```diff
@@ -200,17 +200,11 @@ impl<'tcx> UnifyValue for ConstVarValue<'tcx> {
                 // universe is the minimum of the two universes, because that is
                 // the one which contains the fewest names in scope.
                 let universe = cmp::min(universe1, universe2);
-                (ConstVariableValue::Unknown { universe }, value1.origin.span)
+                (ConstVariableValue::Unknown { universe }, value1.origin)
```

Comment:
And inline the `ConstVarValue { .. }` part here. I don't even think the span should be relevant here, so maybe `DUMMY_SP` would be more semantic.

Reply:
The span is relevant here.  It isn't really deterministic which const infer we use when reporting errors afaict, so we pretty much require all const infer variables to have relevant origin data.

Reply:
Hmm, I was basing my assumption off https://github.com/rust-lang/rust/blob/d1c2815d6aa131182aa93603de215d63c52d0c53/compiler/rustc_infer/src/infer/type_variable.rs#L482-L494
which doesn't keep track of span information, but it's not important either way.